nautobot/dcim/filters/__init__.py

```diff
@@ -1504,13 +1503,13 @@ class InventoryItemFilterSet(BaseFilterSet, DeviceComponentModelFilterSetMixin):
         label="Manufacturer (slug or ID)",
     )
     # TODO: solve https://github.com/nautobot/nautobot/issues/2875 to use this filter correctly
-    child_items = NaturalKeyOrPKMultipleChoiceFilter(
+    children = NaturalKeyOrPKMultipleChoiceFilter(
         queryset=InventoryItem.objects.all(),
         to_field_name="name",
         label="Child items (name or ID)",
     )
-    has_child_items = RelatedMembershipBooleanFilter(
-        field_name="child_items",
+    has_children = RelatedMembershipBooleanFilter(
+        field_name="children",
```

Comment:
Unlike MPTT, django-tree-queries doesn't generally let you change the related-name for the `parent` FK - it has to be `children`.